diff -Naur old/pglogical_output_plugin.c new/pglogical_output_plugin.c
--- old/pglogical_output_plugin.c	2023-10-04 09:54:00
+++ new/pglogical_output_plugin.c	2023-10-24 21:27:44
@@ -189,7 +189,7 @@
 	{
 		int		params_format;
 		bool	started_tx = false;
-		PGLogicalLocalNode *node;
+//		PGLogicalLocalNode *node;
 		MemoryContext oldctx;
 
 		/*
@@ -208,8 +208,8 @@
 			StartTransactionCommand();
 			started_tx = true;
 		}
-		node = get_local_node(false, false);
-		data->local_node_id	= node->node->id;
+//		node = get_local_node(false, false);
+//		data->local_node_id	= node->node->id;
 
 		 /*
 		 * Ideally we'd send the startup message immediately. That way
@@ -669,8 +669,8 @@
 	old = MemoryContextSwitchTo(data->context);
 
 	/* First check the table filter */
-	if (!pglogical_change_filter(data, relation, change, &att_list))
-		goto cleanup;
+//	if (!pglogical_change_filter(data, relation, change, &att_list))
+//		goto cleanup;
 
 	/*
 	 * If the protocol wants to write relation information and the client
